src/components/ProfilePhotoGroup.jsx

```diff
@@ -51,7 +51,7 @@ class ProfilePhotoGroup extends Component {
         ) : (
           <h3>{soc} Execs</h3>
         )}
-        <Row className="mb-3">
+        <Row className="mb-3 ml-0">
```

Comment:
small bug that cuts off left bit of card on about us page

Reply:
Thank you!!!